src/Components/Aspire.MongoDB.Driver/AspireMongoDBDriverExtensions.cs

```diff
@@ -194,6 +194,9 @@ private static MongoClient CreateMongoClient(
         configureClientSettings?.Invoke(clientSettings);
 
         clientSettings.LoggingSettings ??= new LoggingSettings(serviceProvider.GetService<ILoggerFactory>());
+        
+        var aspireVersion = typeof(MongoDBSettings).Assembly.GetName().Version?.ToString();
+        clientSettings.LibraryInfo = new LibraryInfo("aspire", aspireVersion);
```

Comment:
Can you move this up to line 188, right after the clientSettings gets created?

The reasoning is that the app developer should be able to set this LibraryInfo in their `configureClientSettings` callback. Doing it here will overwrite anything the app developer tries configuring.

Reply:
Thanks @eerhardt for raising this, we still want to record the aspire info so I have updated the code to append it if the user has already set the LibraryInfo.